src/validations/rules/ssp.sch

```diff
@@ -1661,6 +1608,46 @@
                 test="current() = $security-objective-levels">[Section B Check 3.10] A FedRAMP SSP must specify an allowed security objective
                 value.</sch:assert>
         </sch:rule>
+        <sch:rule
+            context="o:control-implementation/o:implemented-requirement/o:statement/o:by-component/o:set-parameter"
+            doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.4">
+            <sch:let
+                name="sensitivity-level"
+                value="/ => lv:sensitivity-level() => lv:if-empty-default('')" />
+            <sch:let
+                name="selected-profile"
+                value="$sensitivity-level => lv:profile()" />
+            <sch:let
+                name="profile-catalog-control-params">
+                <array:controls>
+                    <xsl:for-each
+                        select="$selected-profile//o:control">
+                        <o:control>
+                            <xsl:attribute
+                                name="id"
+                                select="@id" />
+                            <xsl:copy-of
+                                select="o:param" />
+                        </o:control>
+                    </xsl:for-each>
+                </array:controls>
+            </sch:let>
+            <sch:let
+                name="controlID"
+                value="../../../@control-id" />
+            <sch:let
+                name="paramID"
+                value="@param-id" />
```

Comment:
This variable is unnecessary - should be `current()/@param-id` in the assertion test.